src/databricks/sql/__init__.py

```diff
@@ -44,7 +44,6 @@ def TimestampFromTicks(ticks):
     return Timestamp(*time.localtime(ticks)[:6])
 
 
-def connect(server_hostname, http_path, access_token, **kwargs):
+def connect(server_hostname, http_path, access_token=None, **kwargs):
```

Comment:
Thinking out loud -- this param being called `access_token` may conflate it with OAuth access tokens, and possibly appear to exclude PAT tokens.

If the earlier assertion is correct that it's used as a required positional param, is this the ideal moment to rename to `auth_token` to avoid the PAT conflation and make consistent with the attached docs/logs?